icu4c/source/common/unicode/urename.h

```diff
@@ -109,10 +109,20 @@
 #define _UTF32LEData U_ICU_ENTRY_POINT_RENAME(_UTF32LEData)
 #define _UTF7Data U_ICU_ENTRY_POINT_RENAME(_UTF7Data)
 #define _UTF8Data U_ICU_ENTRY_POINT_RENAME(_UTF8Data)
+#define __bss_start U_ICU_ENTRY_POINT_RENAME(__bss_start)
+#define _edata U_ICU_ENTRY_POINT_RENAME(_edata)
+#define _end U_ICU_ENTRY_POINT_RENAME(_end)
```

Comment:
short things like _end and _edata aren't ICU API, and shouldn't be getting exported from the ICU libraries or renamed. We should figure out why they're showing up. Anything starting with _ is questionable.